### PR TITLE
add resource for kotlin

### PR DESCRIPTION
**Pull request by Me-Sunil**
## What does this PR do?
Add resource

### Why is this valuable (or not)?
Resource is well structured and easy to navigate. It's Example oriented learning.

### How do we know it's really free?
Yes

### For book lists, is it a book? For course lists, is it a course? etc.
It's an self paced website read.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by sohansai** [comment]
Refer to the contributing guidelines for reference and remember to prefer atomic commits. 

Also, check the status of GitHub Actions and resolve any reported warnings. 

You can find the guidelines [here](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)

**Comment by Me-Sunil** [comment]
Hi @sohansai 
I have fixed the lint issues, can i get review for this.

**Comment by sohansai** [comment]
> Hi @sohansai I have fixed the lint issues, can i get review for this.

Wait for the collaborator's response.

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -1421,6 +1421,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Kotlin Notes for Professionals](https://goalkicker.com/KotlinBook/) - Compiled from StackOverflow documentation (PDF)
 * [Kotlin Official Documentation](https://kotlinlang.org/docs/reference/)
 * [Kotlin Quick Reference](https://kotlin-quick-reference.com) - Alvin Alexander (gitbook)
+* [Learn Kotlin Programming](https://www.programiz.com/kotlin-programming) - Programiz, Kotlin Structured Modular Online Resource
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1424]:
```suggestion
* [Learn Kotlin Programming](https://www.programiz.com/kotlin-programming) - Programiz
```

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**